### PR TITLE
[auth] Adding SAML Configuration documentation

### PR DESCRIPTION
**Pull request by abellotti**
- Added auth/saml.adoc

### CODE REVIEW

**Comment by abellotti** [comment]
WIP - Depends on:

* https://github.com/ManageIQ/manageiq/pull/6603
* https://github.com/ManageIQ/manageiq/pull/7310

**Comment by abellotti** [comment]
@mfeifer please review.  Thanks!!

File: auth/saml.adoc [hunk 1]
```diff
@@ -0,0 +1,224 @@
+
+[[saml]]
+== SAML
+
+External Authentication on the Appliance has been enhanced to support SAML.
+The SAML implementation has been tested with KeyCloak 1.8.
+
+In this guide we will cover how to manually configure a 7.2 Appliance's
```
Comment (mfeifer) [inline auth/saml.adoc]: What project does 7.2 apply to?

Comment (abellotti) [inline auth/saml.adoc]: Ah ok, sorry, this just means RHEL 7.2, but that's just upstream.  I'll update. Thanks.

**Pull request merged**